core/templates/dev/head/pages/exploration_editor/simple_editor_tab/questions/MultipleChoiceEditorDirective.js

```diff
@@ -136,21 +136,30 @@ oppia.directive('multipleChoiceEditor', [
           $scope.saveChoice = function(index, newChoiceValue) {
             if (!newChoiceValue) {
               alertsService.addWarning('Cannot save an empty choice.');
-              return;
+              return {
```

Comment:
Can we introduce a ``Status`` domain object, instead?  It will be clearer, and it can have a success/failure boolean and reason string just like this.